### PR TITLE
PLAT-125067: qa-a11y: Fix spacing for VirtualGridList

### PR DESCRIPTION
**Pull request by juwonjeong**
Enact-DCO-1.0-Signed-off-by: Juwon Jeong (juwon.jeong@lge.com)

### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Fix qa-a11y: VirtualGridList sample to fully display horizontal scrollbar

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Change the spacing between items and scrollbar according to guideline

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
PLAT-125067

### CODE REVIEW

File: samples/sampler/stories/qa/VirtualGridList.js [hunk 1]
```diff
@@ -172,7 +172,7 @@ export const HorizontalVirtualGridList = () => (
 		onScrollStop={action('onScrollStop')}
 		scrollMode={select('scrollMode', prop.scrollModeOption, Config)}
 		spacing={ri.scale(number('spacing', Config, 0))}
-		style={{paddingBottom: ri.unit(ri.scale(36) + 'px', 'rem')}}
+		style={{paddingBottom: ri.unit(ri.scale(24) + 'px', 'rem')}}
```
Comment (jeonghee27) [inline samples/sampler/stories/qa/VirtualGridList.js:175]:
```suggestion
		style={{paddingBottom: ri.unit(ri.scale(90) + 'px', 'rem')}}
```

Comment (juwonjeong) [reply samples/sampler/stories/qa/VirtualGridList.js]: It's fixed.

**Review by jeonghee27 — APPROVED** [review]
LGTM

File: samples/qa-a11y/src/views/VirtualGridList.module.less [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+//
+
+@import '~@enact/sandstone/styles/mixins.less';
+
+.verticalPadding {
+    .padding-start-end(0, 60px);
+}
+
+.horizontalPadding {
+    padding-bottom: 90px;
```
Comment (MikyungKim) [inline samples/qa-a11y/src/views/VirtualGridList.module.less:11]:
```suggestion
    padding-bottom: 36px;
```

Comment (juwonjeong) [reply samples/qa-a11y/src/views/VirtualGridList.module.less]: It's fixed.

File: samples/qa-a11y/src/views/VirtualGridList.module.less [hunk 3]
```diff
@@ -0,0 +1,12 @@
+// VitualGridList.module.less
+//
+
+@import '~@enact/sandstone/styles/mixins.less';
+
+.verticalPadding {
+    .padding-start-end(0, 60px);
```
[the remark this answers is not in the document]
Comment (juwonjeong) [reply samples/qa-a11y/src/views/VirtualGridList.module.less]: It's fixed.

**Review by MikyungKim — APPROVED** [review]
LGTM

**Pull request merged**